From ce6815f6ce4756323736e6d2ec470dc94b7d9486 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Fri, 16 Apr 2021 10:43:34 -0400
Subject: [PATCH] skopeo: bump to 1.2.3 -dev

Pulling in the small set of commits that move us to the
1.2.3-dev cycle:

  5a117bfc Bump c/storage to v1.24.8 and c/image to v5.10.5 for RHEL 8.4 in release-1.2
  560a34af Bump to v1.2.3-dev
  b94b7dc0 Bump to Skopeo v1.2.2
  f78bf42c Bump c/common c/image and c/storage to latest
  b4210c0b Fix gating test in release-1.2 port #1169

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 recipes-containers/skopeo/skopeo_git.bb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/recipes-containers/skopeo/skopeo_git.bb b/recipes-containers/skopeo/skopeo_git.bb
index be72dc1..05baefb 100644
--- a/recipes-containers/skopeo/skopeo_git.bb
+++ b/recipes-containers/skopeo/skopeo_git.bb
@@ -25,8 +25,8 @@ SRC_URI = " \
     file://registries.conf \
 "
 
-SRCREV = "6c0e35a50c404c88eba84b4c611e6ea06ed33a44"
-PV = "v1.2.2-dev+git${SRCPV}"
+SRCREV = "e7880c4a8991966f16e367f085d42375ad70197e"
+PV = "v1.2.3-dev+git${SRCPV}"
 GO_IMPORT = "import"
 
 S = "${WORKDIR}/git"
-- 
2.25.1

